release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -578,6 +578,24 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-13-oc-mirror-openshift-endpoint"]
+=== The oc-mirror plugin now retrieves graph data container images from an OpenShift API endpoint
+
+The oc-mirror OpenShift CLI (`oc`) plugin now downloads the graph-data tarball from an OpenShift API endpoint instead of downloading the entire graph-data repository from GitHub. Retrieving this data from Red Hat instead of an external vendor is more suitable for users with stringent security and compliance restrictions on external content.
+
+The data that the oc-mirror plugin downloads will now exclude content that is in the graph-data repository but not needed by the OpenShift Update Service. The container also uses UBI Micro as the base image instead of UBI, resulting in a container image that is about 85% smaller than before.
```

Comment:
Removing future tense. Also, is the specific 85% critical to mention?

```suggestion
The data that the oc-mirror plugin downloads now excludes content that is in the graph-data repository but not needed by the OpenShift Update Service. The container also uses UBI Micro as the base image instead of UBI, resulting in a container image that is about 85% smaller than before.
```

Reply:
The specific value is honestly there because it seemed like specifying how dramatically smaller the file would be sounded "better" than just saying "resulting in a much smaller container image size." 

I'm happy to remove the specific value if you would rather it removed, the choice to include it was mostly subjective.

Reply:
I mostly just worry about giving exact values (though I realize it does say "about"), just in case customers complain that their decrease in value is different. But I don't really have anything to point to saying you can't :shrug:

Reply:
Technically users can look up the sizes for UBI and UBI Micro and make the comparison themselves, so I'll just substitute "about 85%" with "significantly" and limit the possibility for customer complaints.